Platform/Microsoft/OpteeClientPkg/Drivers/AuthVarOpteeRuntimeDxe/Measurement.c

```diff
@@ -0,0 +1,252 @@
+/** @file
+Measure TrEE required variable.
+Copyright (c) 2013 - 2014, Intel Corporation. All rights reserved.<BR>
+This program and the accompanying materials
+are licensed and made available under the terms and conditions of the BSD License
+which accompanies this distribution.  The full text of the license may be found at
+http://opensource.org/licenses/bsd-license.php
+THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+**/
+
+#include <PiDxe.h>
+#include <Guid/ImageAuthentication.h>
+#include <IndustryStandard/UefiTcgPlatform.h>
+#include <Protocol/TrEEProtocol.h>
+#include <Protocol/Variable.h>
+
+#include <Library/UefiBootServicesTableLib.h>
+#include <Library/UefiRuntimeServicesTableLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseLib.h>
+#include <Library/TpmMeasurementLib.h>
+
+typedef struct {
+  CHAR16                                 *VariableName;
+  EFI_GUID                               *VendorGuid;
+} VARIABLE_TYPE;
+
+VARIABLE_TYPE  mVariableType[] = {
+  { EFI_SECURE_BOOT_MODE_NAME,    &gEfiGlobalVariableGuid },
+  { EFI_PLATFORM_KEY_NAME,        &gEfiGlobalVariableGuid },
+  { EFI_KEY_EXCHANGE_KEY_NAME,    &gEfiGlobalVariableGuid },
+  { EFI_IMAGE_SECURITY_DATABASE,  &gEfiImageSecurityDatabaseGuid },
+  { EFI_IMAGE_SECURITY_DATABASE1, &gEfiImageSecurityDatabaseGuid },
+};
+
+/**
+This function will return if this variable is SecureBootPolicy Variable.
+@param[in]  VariableName      A Null-terminated string that is the name of the vendor's variable.
+@param[in]  VendorGuid        A unique identifier for the vendor.
+@retval TRUE                  This is SecureBootPolicy Variable
+@retval FALSE                 This is not SecureBootPolicy Variable
+**/
+BOOLEAN
+IsSecureBootPolicyVariable(
+  IN CHAR16                                 *VariableName,
+  IN EFI_GUID                               *VendorGuid
+)
```

Comment:
nit - needs 2 spaces

```suggestion
  )
```